partitioned-table.md

```diff
@@ -753,76 +756,166 @@ In addition, there are limitations on the compatibility of `EXCHANGE PARTITION`
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members ADD PARTITION (PARTITION p1990 VALUES LESS THAN (2000));
+```
+
+```
+ERROR 1493 (HY000): VALUES LESS THAN value must be strictly increasing for each partition
 ```
-Query OK, 0 rows affected (0.03 sec)
+
+#### Reorganize partitions
+
+Split a partition:
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members REORGANIZE PARTITION `p1990to2010` INTO
+(PARTITION p1990 VALUES LESS THAN (2000),
+ PARTITION p2000 VALUES LESS THAN (2010),
+ PARTITION p2010 VALUES LESS THAN (2020),
+ PARTITION p2020 VALUES LESS THAN (2030),
+ PARTITION pMax VALUES LESS THAN (MAXVALUE));
+
+ALTER TABLE member_level REORGANIZE PARTITION l5_6 INTO
+(PARTITION l5 VALUES IN (5),
+ PARTITION l6 VALUES IN (6));
 ```
 
-Empty a partition:
+Merge partitions:
 
 {{< copyable "sql" >}}
 
 ```sql
-ALTER TABLE members TRUNCATE PARTITION p1;
+ALTER TABLE members REORGANIZE PARTITION pBefore1950,p1950 INTO (PARTITION pBefore1960 VALUES LESS THAN (1960));
+
+ALTER TABLE member_level REORGANIZE PARTITION l1,l2 INTO (PARTITION l1_2 VALUES IN (1,2));
 ```
 
+Change the partitioning scheme definition:
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members REORGANIZE PARTITION pBefore1960,p1960,p1970,p1980,p1990,p2000,p2010,p2020,pMax INTO
+(PARTITION p1800 VALUES LESS THAN (1900),
+ PARTITION p1900 VALUES LESS THAN (2000),
+ PARTITION p2000 VALUES LESS THAN (2100));
+
+ALTER TABLE member_level REORGANIZE PARTITION l1_2,l3,l4,l5,l6 INTO
+(PARTITION lOdd VALUES IN (1,3,5),
+ PARTITION lEven VALUES IN (2,4,6));
 ```
-Query OK, 0 rows affected (0.03 sec)
+
+Reorganizing partitions (including merging or splitting partitions) changes the listed partitions into a new set of partition definitions. It cannot change the type of partitioning (for example, change the List type to the Range type, or change the Range COLUMNS type to the Range type).
+
+For Range partitions, you can reorganize only adjacent partitions.
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members REORGANIZE PARTITION p1800,p2000 INTO (PARTITION p2000 VALUES LESS THAN (2100));
 ```
 
-> **Note:**
->
-> `ALTER TABLE ... REORGANIZE PARTITION` is currently unsupported in TiDB.
+```
+ERROR 8200 (HY000): Unsupported REORGANIZE PARTITION of RANGE; not adjacent partitions
+```
 
-Add a partition:
+For a Range partitioned table, to modify the end of the range, the new end defined in `VALUES LESS THAN` must cover the existing rows in the last partition. If the end is changed and existing rows no longer fit, an error is reported:
 
 {{< copyable "sql" >}}
 
 ```sql
-ALTER TABLE members ADD PARTITION (PARTITION p3 VALUES LESS THAN (2010));
+INSERT INTO members VALUES (313, "John", "Doe", "2022-11-22", NULL);
+ALTER TABLE members REORGANIZE PARTITION p2000 INTO (PARTITION p2000 VALUES LESS THAN (2050)); -- succeeds, since it covers the existing row
```

Comment:
```suggestion
ALTER TABLE members REORGANIZE PARTITION p2000 INTO (PARTITION p2000 VALUES LESS THAN (2050)); -- succeeds, since it covers the existing rows
```